app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,23 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
```

Comment:
can you please summarize here the difference between `AuthenticationParameters` and `AuthenticatorInput`? to me they mean the same thing.

Reply:
Talked about this in sunday